### PR TITLE
[WIP] Add Redis as MQ Adapter

### PR DESCRIPTION
**Pull request by kandy**
### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by kandy** [comment]
@magento run all tests

**Comment by tanghongtanghong** [comment]
test0125

File: app/code/Magento/Catalog/etc/queue_consumer.xml [hunk 1]
```diff
@@ -6,6 +6,6 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework-message-queue:etc/consumer.xsd">
-    <consumer name="product_action_attribute.update" queue="product_action_attribute.update" connection="db" maxMessages="5000" consumerInstance="Magento\Framework\MessageQueue\Consumer" handler="Magento\Catalog\Model\Attribute\Backend\Consumer::process" />
-    <consumer name="product_action_attribute.website.update" queue="product_action_attribute.website.update" connection="db" maxMessages="5000" consumerInstance="Magento\Framework\MessageQueue\Consumer" handler="Magento\Catalog\Model\Attribute\Backend\ConsumerWebsiteAssign::process" />
```
Comment (tanghongtanghong) [inline app/code/Magento/Catalog/etc/queue_consumer.xml:10]: test123

**Review by tanghongtanghong — APPROVED** [review]
Approve

File: app/code/Magento/Catalog/etc/queue_consumer.xml [hunk 2]
```diff
@@ -6,6 +6,6 @@
...
-</config>
\ No newline at end of file
+    <consumer name="product_action_attribute.update"  handler="Magento\Catalog\Model\Attribute\Backend\Consumer::process" />
+    <consumer name="product_action_attribute.update"  handler="Magento\Catalog\Model\Attribute\Backend\ConsumerWebsiteAssign::process" />
```
Comment (tanghongtanghong) [inline app/code/Magento/Catalog/etc/queue_consumer.xml]: sorry

**Review by tanghongtanghong — COMMENTED** [review]
sorry

**Comment by kandy** [comment]
@magento run all tests

**Comment by kandy** [comment]
@magento run Integration Tests

**Comment by kandy** [comment]
@magento run Integration Tests

**Comment by kandy** [comment]
@magento run Integration Tests

**Comment by kandy** [comment]
@magento run Integration Tests

**Comment by kandy** [comment]
@magento run Integration Tests

**Comment by kandy** [comment]
@magento run Integration Tests

**Pull request closed without merging**